app/models/manifest.rb

```diff
@@ -0,0 +1,139 @@
+class Manifest < CacheableResource
+  DEFAULT_SETTINGS = {
+    'filename' => 'manifest.json',
+    'mime_type' => 'application/json'
+  }
+  METS_NS = { 'mets' => 'http://www.loc.gov/METS/' }
+  THUMBNAIL_OPTS = {
+    version: 2,
+    region: 'full',
+    size: '!256,256',
+    rotation: 0,
+    quality: 'native',
+    format: 'jpg'
+  }
+  attr_reader :route_helper
+  def initialize(id_or_fedora_obj, route_helper)
+    super(id_or_fedora_obj)
+    @route_helper = route_helper
+  end
+
+  def queue_manifest_generation(queue_name = Derivativo::Queue::HIGH)
+    base_url = route_helper.iiif_id_url(id: 'do_not_use', version: THUMBNAIL_OPTS[:version])
```

Comment:
If I'm understanding the code correctly, only the host, protocol, and port portions of this URL are important, right?  In that case, can we chop off the unhelpful portion of the URL before we pass it to the CreateManifestJob on the line beneath this one?  That way we won't have lots of "do_not_use" strings in the arguments of our job queue, which could seem confusing/incorrect later on (even though they wouldn't have any real effect).

Reply:
I think that should do it.

Reply:
Thank you for all of your work!  Approving the PR now!